controller/users_controller.go

```diff
@@ -0,0 +1,108 @@
+package controller
+
+import (
+	"mytodoapp/data1/request"
+	"mytodoapp/data1/response"
+	"mytodoapp/helper"
+	"mytodoapp/service"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"github.com/rs/zerolog/log"
+)
+
+type UsersController struct {
+	usersService service.UsersService
+}
+
+func NewUsersController(service service.UsersService) *UsersController {
+	return &UsersController{
+		usersService: service,
+	}
+}
+
+func (controller *UsersController) Create(ctx *gin.Context) {
+	log.Info().Msg("create users")
+	CreateUsersRequest := request.CreateUsersRequest{}
+	err := ctx.ShouldBindJSON(&CreateUsersRequest)
+	helper.ErrorPanic(err)
```

Comment:
instead of panic lets return Bad request as the status code and the error message in response